apps/server/src/modules/user-login-migration/loggable/user-login-migration-mandatory.loggable.ts

```diff
@@ -4,8 +4,8 @@ import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from
 export class UserLoginMigrationMandatoryLoggable implements Loggable {
 	constructor(
 		private readonly userId: EntityId,
-		private readonly userLoginMigrationId: EntityId,
-		private readonly mandatory: boolean
+		private readonly mandatory: boolean,
+		private readonly userLoginMigrationId?: EntityId
```

Comment:
I think `userLoginMigrationId: EntityId | undefined` in the middle would be better than `userLoginMigrationId?: EntityId` at the end, because when calling the constructor you have to intentionally set undefined. When you use the optional parameter at the end then you can forget to add the parameter when you implement a new call.